tests/expected_tests.cc

```diff
@@ -0,0 +1,39 @@
+#include "gtest/gtest.h"
+#include "stout/expected.h"
+
+stout::expected<int, std::string> divide(int divident, int divider) {
+  if (!divider) {
```

Comment:
Generally, I'd avoid implicit conversions to bool: https://abseil.io/tips/141

So here: `if (divider == 0)`